docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -91,6 +91,10 @@ If the view defined in `openTo` is not the first view, then the views before wil
 (for example view the default behaviors, the `year` is only accessible when clicking on the toolbar).
 :::
 
+Intended for use with the standalone year picker, the `yearsReversed` prop can be set to show the maximum year first and minimum year at the end of the list.
```

Comment:
> WDYT about moving this example into the [DateCalendar page](https://mui.com/x/react-date-pickers/date-calendar/) instead?

Definitely

Reply:
Sounds good!

My thinking was to keep the scope narrow and I was thinking it could be a problem for accessibility if the years are in a different order than the days and months which will probably never have a case that needs them to be reversed.

But if the year reversal is useful for other calendar cases then definitely better to give it more visibility in the examples there. 👌

Reply:
a11y is a great consideration, but I'm not sure if it is a big problem.
Users would implicitly understand the order of the years navigating the calendar.
If it would be signaled as a problem, we could then adjust the documentation warning about this potentially unclear behavior. 👌

Reply:
I added a new demo with the calendar view with years reversed, but I think the picker demo is quite good also. Is it ok now with both? They are quite small so hopefully doesn't detract too much from the other demos.

Reply:
I think that both demos can co-exist. 👍